docs/source/registries.rst

```diff
@@ -0,0 +1,24 @@
+Registries
+==========
+
+All models in Curated Transformers are added to a registry. Each auto class uses
+a registry to query which models are available. This mechanism allows
+third-party models to hook into the auto classes. This makes it possible to use
+construction methods such as ``AutoModel.from_hf_hub`` with third-party models.
```

Comment:
Could we make `from_hf_hub` a hyperlink?

Reply:
We can't because we don't expose `AutoModel` in the API docs, but maybe we should link to a concrete implementation here, is does say _such as_ after all.

Reply:
Linked.